mount/mount.cvmfs.cc

```diff
@@ -695,7 +695,11 @@ int main(int argc, char **argv) {
       }
     }
     ended = (waitpid(pid_cvmfs, &status, WNOHANG) == pid_cvmfs);
```

Comment:
Why set ended inside the loop when it is only checked outside?  I guess this could then be collapsed to just always invoke waitpid afterward, as it was done before #3144.

Reply:
But `ended` is used in the `while` condition of the loop